src/com/estafet/learning/US008/Order.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+public abstract class Order {
+    public static final String ORDER_NAME = "ORR";
+    public String orderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+    private static final int invoice = 1;
+    public static final String color = "green";
+
+    public Order(){
+    }
+
+    public Order(String orderNumber, String ClientDetails){
```

Comment:
arguments, variables names should start with a small letter

Reply:
changed.